rebasehelper/cli.py

```diff
@@ -90,20 +90,29 @@ def build_parser():
             action="store_true",
             help="be more verbose (recommended)"
         )
-        parser.add_argument(
+        patch_build_cmp_only_group = parser.add_mutually_exclusive_group()
```

Comment:
I think simple `group` would suffice, since it is not referenced anywhere and so it doesn't have to be unique.